packages/editor/src/browser/editor.ts

```diff
@@ -40,7 +40,8 @@ export interface TextEditor extends Disposable, TextEditorSelection {
     readonly onFocusChanged: Event<boolean>;
 
     revealPosition(position: Position): void;
```

Comment:
Should we add options to `revealPosition` as well? fyi: looking at monaco API, one can specify whether a position should be revealed vertically or horizontally.

Reply:
Please, have a look.